source/help/atom_specifications.md

```diff
@@ -73,6 +73,11 @@ Subject classifications. Articles can be classified under more than one category
 <category term="math.AP"/>
 <category term="math.OC"/>
 ```
+**&lt;published&gt;**
+Publish date for the update on the article. For new announcements this is the date the paper was officially announced on arXiv, for all other its the date of the official announcement of the update.  
```

Comment:
I disagree with framing it this way. We're not using the term "published" elsewhere.